src/foam/nanos/cron/Cron.js

```diff
@@ -8,8 +8,108 @@ foam.CLASS({
   package: 'foam.nanos.cron',
   name: 'Cron',
   extends: 'foam.nanos.script.Script',
-
+  javaImports: [
+    'java.util.Date',
+    'java.util.Calendar'
+  ],
+  documentation: 'FOAM class that models a Cron script',
   properties: [
-    // TODO: Add scheduling properties
+    {
+      class: 'Int',
+      name: 'minute',
+      value: -1,
+      documentation: `Minute to execute script.
+          Ranges from 0 - 59. -1 for wildcard`
+    },
+    {
+      class: 'Int',
+      name: 'hour',
+      value: -1,
+      documentation: `Hour to execute script.
+          Ranges from 0 - 23. -1 for wildcard`
+    },
+    {
+      class: 'Int',
+      name: 'dayOfMonth',
+      value: -1,
+      documentation: `Day of Month to execute script.
+          Ranges from 1 - 31. -1 for wildcard`
+    },
+    {
+      class: 'Int',
+      name: 'month',
+      value: -1,
+      documentation: `Month to execute script.
+          Ranges from 1 - 12. -1 for wildcard`
+    },
+    {
+      class: 'Int',
+      name: 'dayOfWeek',
+      value: -1,
+      documentation: `Day of week to execute script.
+          Ranges from 0 - 6, where 0 is Sunday. -1 for wildcard`
+    },
+    {
+      class: 'DateTime',
+      name: 'scheduledTime',
+      documentation: `Scheduled time to run Cron script.`,
+      javaFactory: function() {
```

Comment:
Instead of javaFatory: function() { /*  code goes here */ }, you can use
javaFactory: `
  code goes here
`